### ISSUE TITLE
Improve Issue transaction logic for cryptocurrency-advanced [ECR-1132]

### ISSUE THREAD

**Issue by DarkEld3r** [exonum/exonum#633]
Currently `Issue` [transaction](https://github.com/exonum/exonum/blob/master/examples/cryptocurrency-advanced/backend/src/transactions.rs#L71) can be sent by anyone. Its logic should be updated, so that way that this transaction must be signed by the special key stored in the config.

**Comment by jgollhardt** [comment]
The docs for verifying transactions specify that "This method should not use external data, that is, it must be a pure function."

Wouldn't this require reading a public key from outside the context of the transaction?

I'm probably just missing something.

**Comment by DarkEld3r** [comment]
@jgollhardt You are right, technically `verify` doesn't have access to the database, but the key (or other data) can be stored inside of the global (`lazy_static`) variable that can be accessed inside the `verify` method. It isn't an elegant solution, but we use it in several (not public) services.

After #1002 we don't have the `verify` function anymore, so documentation is outdated (it will be updated soon). Now all checks should be done inside of the `execute` method.

**Comment by jgollhardt** [comment]
Ah, that makes sense. Thank you @DarkEld3r !
